Gemfile

```diff
@@ -68,6 +68,8 @@ else
   gem "groonga-client-model"
 end
 
+gem "importmap-rails", "~> 2.0"
```

Comment:
Do we need to specify a version?

Reply:
Sorry, I forgot to mention it. This change is auto generated by executing the following command.
I think it's better to accept changes by minor versions because major version up accept the breaking changes.

```
$ ./bin/bundle add importmap-rails
```
ref: https://github.com/rails/importmap-rails?tab=readme-ov-file#installation